battleship/battleship-multi.py

```diff
@@ -0,0 +1,320 @@
+    sleep(5)
+
+    # Clear system output
+    os.system('cls' if os.name == 'nt' else 'clear')
+
+    return playerName
+
+
+def playerTurn(shipNames, opponentGrid, opponentShipsAlive, opponentShipAliveCoords, opponentShipSunkCoords, playerName):
+    '''Execute player turn'''
+
+    print("{}'S TURN!".format(playerName))
+    printGrid(opponentGrid, False, playerName)
+    sunkShip = False
+
+    # Player input and logic
+    while True:
+        # Get attack coordinate and check validity
+        attack = input('Coordinate for your attack this turn? ')
+        attackCoords = convert(attack)
+        if attackCoords is None:
+            print(
+                'Invalid input. Examples of proper coordinates: a0, B9, c2. Please try again.')
+            continue
+        elif opponentGrid[attackCoords[0]][attackCoords[1]] != ' ':
+            print('You have already attacked this coordinate. Please try again.')
+            continue
+
+        # Check if attack hit a ship and update grid
+        shipIndex = hitShip(attackCoords, opponentShipAliveCoords)
+        if shipIndex == -1:
+            print("Miss!")
+            opponentGrid[attackCoords[0]][attackCoords[1]] = '-'
+            break
+        else:
+            print("Hit!")
+            opponentGrid[attackCoords[0]][attackCoords[1]] = 'o'
+            # Remove attack coordinate from alive set to sunk set
+            opponentShipAliveCoords[shipIndex].remove(attackCoords)
+            opponentShipSunkCoords[shipIndex].add(attackCoords)
+            # Check if alive set is empty (meaning ship is sunk)
+            if len(opponentShipAliveCoords[shipIndex]) == 0:
+                sunkShip = True
+                print("You have sunk your opponent's " +
+                      shipNames[shipIndex] + "!")
+                opponentShipsAlive -= 1
+                for coords in opponentShipSunkCoords[shipIndex]:
+                    opponentGrid[coords[0]][coords[1]] = 'x'
+            break
+
+    # Print grid again if ship was sunk
+    if sunkShip:
+        printGrid(opponentGrid, False, playerName)
+    print("END TURN!")
+    print("--------------------------------------------------")
+    if sunkShip:
+        sleep(3)
+    sleep(2)
+    return opponentShipsAlive
+
+
+def printGrid(grid, showShips, playerName):
+    '''Print the specified grid'''
+
+    # Set title based on isPlayerGrid
+    if showShips:
+        print("\n{}'S SHIPS".format(playerName))
+    else:
+        print("\n{}'S ATTACKS".format(playerName))
+
+    # Print grid
+    for i in range(-1, 10):
+        for j in range(-1, 10):
+            # Print first row of letters
+            if i == -1:
+                if j == -1:
+                    print(" ", end=" ")
+                elif j == 9:
+                    print(chr(j+48))
+                else:
+                    print(chr(j+48), end=" ")
```

Comment:
chr(j+48) can just be replaced with j
```suggestion
                    print(j)
                else:
                    print(j, end=" ")
```

Reply:
Oops yes that would be much better